Dockerfile

```diff
@@ -0,0 +1,65 @@
+# First, get the elixir dependencies within an elixir container
+FROM hexpm/elixir:1.11.3-erlang-23.2.5-alpine-3.13.1 AS elixir-builder
+
+ENV LANG="C.UTF-8" MIX_ENV=prod
+
+WORKDIR /root
+ADD . .
+
+# Install hex, rebar, and deps
+RUN mix local.hex --force && \
+  mix local.rebar --force && \
+  mix do deps.get --only prod
+
+# Next, build the frontend assets within a node.js container
+FROM node:14.15-alpine as assets-builder
+
+WORKDIR /root
+ADD . .
+
+# Copy in elixir deps required to build node modules for phoenix
+COPY --from=elixir-builder /root/deps ./deps
+
+RUN npm --prefix assets ci
+RUN npm --prefix assets run deploy
+
+# Now, build the application back in the elixir container
+FROM elixir-builder as app-builder
+
+ENV LANG="C.UTF-8" MIX_ENV=prod
+
+WORKDIR /root
+
+# Add frontend assets compiled in node container, required by phx.digest
+COPY --from=assets-builder /root/priv/static ./priv/static
+
+RUN mix do compile --force, phx.digest, release
+
+# Finally, use a Debian container for the runtime environment
+FROM alpine:latest
```

Comment:
- should update the comment to say "Alpine"
- You should pin this to version `3.13.1` since that's the version used by the Elixir container.